openshift/template.yaml

```diff
@@ -97,6 +97,8 @@ parameters:
   value: '3m'
 - name: SUPPORT_L2
   value: "true"
+- name: PUBLIC_CONTAINER_REGISTRIES
```

Comment:
Did you change the deployment scripts in app-sre?

Reply:
Working on it. I'm not going to release this PR before making sure all the environments continue to work.